examples/src/main/kotlin/io/aexp/bucketlist/examples/prfileoccurrences/ExportPrFileOccurrencesData.kt

```diff
@@ -0,0 +1,150 @@
+package io.aexp.bucketlist.examples.prfileoccurrences
+
+import com.google.common.collect.HashMultiset
+import com.google.common.collect.Multiset
+import com.google.common.collect.Multisets
+import com.opencsv.CSVWriter
+import io.aexp.bucketlist.BucketListClient
+import io.aexp.bucketlist.data.PullRequestState
+import io.aexp.bucketlist.examples.getBitBucketClient
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import rx.Observable
+import rx.schedulers.Schedulers
+import java.io.FileWriter
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.util.ArrayList
+
+/**
+ * This example downloads diff data for PRs in the specified repo and writes occurrence counts for files that appear in
+ * at least one PR (sorted by occurrence count).
+ *
+ * We wanted to better inform our decision of when to refactor different classes by identifying which were more highly
+ * correlated with bugs. So this determines in how many PRs across a given date range was each file changed. With the
+ * 'branch filter substring', we were able to reduce the set to only ones including our 'bugfix' prefix.
+ *
+ * This example is purely a data generator because we found that inspecting the top of the resulting list provided ample
```

Comment:
I've never been terribly happy with "data generator" as a short description of "thing that outputs structured data instead of a pretty picture". I'm blanking on better terms, but still :(